tests/cluster/delta/deltaspec/test_dbspec.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from spetlr import Configurator
+from spetlr.configurator.sql.parse_sql import parse_sql_code_to_config
+from spetlr.deltaspec.DeltaDatabaseSpec import DeltaDatabaseSpec
+from spetlr.spark import Spark
+
+
+class TestDbSpec(unittest.TestCase):
```

Comment:
docstring

Reply:
I think docstrings in unittest are a bit overkill